manifests/metricbeat.yaml

```diff
@@ -94,7 +98,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: metricbeat
-        image: docker.elastic.co/beats/metricbeat:6.3.0
+        image: ruflin/mb-snapshot
```

Comment:
I think we should wait for `6.4` before merging this

Reply:
Now that we got snapshot docker images we should use ` docker.elastic.co/beats/metricbeat:7.0.0-alpha1-SNAPSHOT` here. Same for the other Beats.

Reply:
I opened https://github.com/exekias/beats-k8s-dev/pull/3 for this.